packages/venia-ui/lib/components/MiniCart/editItem.js

```diff
@@ -38,7 +38,10 @@ const EditItem = props => {
             return;
         }
 
-        const fetchItemOptions = async () => {
+        // We know the item has options, but we don't know the
+        // other possible values those options could take on.
+        // Fetch them now.
+        const fetchItemVariants = async () => {
```

Comment:
I renamed this function because it was a frequent point of confusion "wait, doesn't the item already have options?". This makes it more clear what we're actually fetching.